apis/metal3.io/v1alpha1/baremetalhost_types.go

```diff
@@ -660,6 +660,18 @@ type RebootAnnotationArguments struct {
 	Mode RebootMode `json:"mode"`
 }
 
+type DetachedDeleteAction string
+
+const (
+	DetachedDeleteActionDelay  = "delay"
+	DetachedDeleteActionDelete = "delete"
+)
+
+type DetachedAnnotationArguments struct {
+	// DeprovisionAction indicates the desired deprovition logic when the detached annotation is present
```

Comment:
And now it's my turn to spot a typo: s/Deprovision/Delete/

Reply:
I see your `s/Deprovision/Delete/` and raise you `s/deprovition/delete/`

Reply:
:man_facepalming: